### ISSUE TITLE
Design visuals for the Sending Bitcoin section

### ISSUE THREAD

**Issue by pavlenex** [BitcoinDesign/Guide#207]
In #181 we've added the new page, `Send Bitcoin`.

https://bitcoin.design/guide/payments/send/

This page needs a header image and we also need an illustration for the Confirmations section.

Whoever picks this issue up can ask @johnsBeharry for the pointers.

**Comment by pavlenex** [comment]
Good quesiton @sbddesign.
@johnsBeharry Do we still need a confirmation image, it seems we already have one, do you remember what we wanted to do with it?

@sbddesign Well for starters let's go with a header image until we figure out what I meant with the "confirmation" illustration lol 😄

**Comment by johnsBeharry** [comment]
@sbddesign the confirmation image is just lacking a bit. Doesn't quite fit in with the rest. I guess for this one its a more nice to have.

There is no header image though - Christoph mentioned this icon style approach as seen on the [personal finance](https://bitcoin.design/guide/getting-started/personal-finance/) header which would be cool for this page.

**Comment by sbddesign** [comment]
I created a header image for the page. I posted it in this Slack thread where we can discuss it:
https://bitcoindesign.slack.com/archives/C015856BDME/p1619122777201800

As for the "Confirmations" section, if somebody could provide me with access to the Figma files used to create the rest of the graphics on this page, that would help me a lot with keeping the style consistent.

**Comment by GBKS** [comment]
The on-page images are done, merged and live.

**Comment by pavlenex** [comment]
I guess we're waiting for the header image that's on @sbddesign's TO-DO before we can close this one.

**Comment by pavlenex** [comment]
Done in #296
